apps/src/lib/ui/SchoolInfoInterstitial.jsx

```diff
@@ -367,18 +372,20 @@ export default class SchoolInfoInterstitial extends React.Component {
           </div>
           <div style={styles.bottom}>
             <Button
-              onClick={this.dismissSchoolInfoForm}
+              onClick={() => this.dismissSchoolInfoForm()}
```

Comment:
@islemaster Hi Brad, I am not sure why this is necessary to get the unit test (closes the dialog when the dismiss button is clicked).  Is it the way the onClick prop is passed down from the button component?  I tried this out and it worked but it doesn't seem right.